libs/state/actions/src/lib/inject.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  ChangeDetectorRef,
+  ErrorHandler,
+  ViewRef,
+  inject,
+} from '@angular/core';
+import { RxActionFactory } from './actions.factory';
+
+export function injectRxActionFactory<
+  Actions extends object
+>(): RxActionFactory<Actions> {
+  const errorHandler = inject(ErrorHandler, { optional: true }) ?? undefined;
+  const rxActionFactory = new RxActionFactory<Actions>(errorHandler);
+  /**
+   * @todo: Use DestroyRef instead when upgrading to Angular 16
+   */
```

Comment:
imo we should not introduce this with v15, because it will be hard to communicate about the component level destroy hook.